.github/workflows/check-failed-workflows.yaml

```diff
@@ -0,0 +1,48 @@
+---
+name: Check failed auto-pull-request workflows
+on:
+  schedule:
+    # Every hour
+    - cron: '0 */1 * * *'
```

Comment:
per hour could be too expensive. 

Could we start with per day job?

cc @glazychev-art